dffml/df/base.py

```diff
@@ -193,6 +193,20 @@ def wrap(func):
         if not "conditions" in kwargs:
             kwargs["conditions"] = []
 
+        primitive_types = ["int", "float", "str", "bool"]
+
+        if not "inputs" in kwargs:
+            sig = str(inspect.signature(func))
```

Comment:
Let's not parse the string, look on line [222](https://github.com/intel/dffml/pull/594/files#diff-9433e166ad553c4951da0eb1beb170c8R222) to see how to get each parameter.

Reply:
Done